src/InternflateDekoratør.tsx

```diff
@@ -2,62 +2,93 @@ import React, { FunctionComponent } from 'react';
+    markup?: Markup; // Egen HTML
+    showEnheter: boolean; // Vis enheter
+    showSearchArea: boolean; // Vis søkefelt
+    showHotkeys: boolean; // Vis hurtigtaster
+    environment: Environment; // Miljø som skal brukes.
+    urlFormat: UrlFormat; // URL format
+    proxy?: string | undefined; // Manuell overstyring av urlene til BFFs. Gjør alle kall til relativt path hvis true, og bruker verdien som domene om satt til en string. Default: false
 }
 
 export interface Markup {
     etterSokefelt?: string; // Gir muligheten for sende inn egen html som blir en del av dekoratøren
 }
 
-// Fnr/Enhet-konfiguration støttet både `Controlled` og `Uncontrolled` operasjon.
-// Ved bruk av `Controlled` må konsument-applikasjonen selv ta ansvar for oppdatering av `value` etter enhver `onChange`
-// Dette er i motsetning til `Uncontrolled`, hvor dette håndteres av dekoratøren. Og alt konsument-applikasjonen trenger å gjøre er å følge med på `onChange`.
-export interface ControlledContextvalue<T> extends BaseContextvalue<T> {
-    value: string | null;
-}
-
-export interface UncontrolledContextvalue<T> extends BaseContextvalue<T> {
-    initialValue: string | null;
+export interface Enhet {
+    readonly enhetId: string;
+    readonly navn: string;
 }
 
-export interface BaseContextvalue<T> {
-    display: T;
+// Miljø
+export type Environment = 'q0' | 'q1' | 'q2' | 'q3' | 'q4' | 'prod' | 'local' | 'mock';
 
-    onChange(value: string | null): void;
+export type UrlFormat = 'LOCAL' | 'NAV_NO' | 'ANSATT'; // UrlFormat. Brukes om proxy ikke er satt & i url til websocket.
 
-    skipModal?: boolean;
-    ignoreWsEvents?: boolean;
+export interface HotkeyObject {
+    char: string;
+    altKey?: boolean;
+    ctrlKey?: boolean;
+    metaKey?: boolean;
+    shiftKey?: boolean;
 }
 
-export type Contextvalue<T> = ControlledContextvalue<T> | UncontrolledContextvalue<T>;
+export interface HotkeyDescription {
+    key: HotkeyObject;
+    description: string;
+    forceOverride?: boolean;
+}
 
-export enum EnhetDisplay {
-    ENHET = 'ENHET',
-    ENHET_VALG = 'ENHET_VALG',
+export interface ActionHotKey extends HotkeyDescription {
+    action(event: KeyboardEvent): void;
 }
 
-export enum FnrDisplay {
-    SOKEFELT = 'SOKEFELT',
+export interface DocumentingHotKey extends HotkeyDescription {
+    documentationOnly: boolean;
 }
 
-export type EnhetContextvalue = Contextvalue<EnhetDisplay>;
-export type FnrContextvalue = Contextvalue<FnrDisplay>;
+export type Hotkey = ActionHotKey | DocumentingHotKey;
+
+const InternflateDecorator = NAVSPA.importer<DecoratorProps>('internarbeidsflate-decorator-v3');
 
-const InternflateDecorator = NAVSPA.importer<DecoratorProps>('internarbeidsflatefs');
+const env = () => {
+    if (window.location.hostname.includes('localhost')) {
+        return 'local';
+    }
+    if (window.location.hostname.includes('intern.dev.nav.no')) {
+        return 'dev';
+    }
+    return 'prod';
+};
```

Comment:
Bruk vite variabler?

Reply:
Enig i at vi på sikt bør gjøre dette; enn så lenge bygger vi likt for prod og dev, så da må det nesten bli slik!